.github/workflows/build-apps.yml

```diff
@@ -130,13 +130,19 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - os: macos-14
+          - os: namespace-profile-macos-6-cores
             platform: mac
-          - os: windows-2022
+            version: macos-14
+          - os: namespace-profile-windows-4-cores
             platform: win
+            version: windows-2022
           - os: ubuntu-22.04
             platform: linux
+            version: ubuntu-22.04
     runs-on: ${{ matrix.os }}
+    # The version prop is only used here to maintain job name consistency
+    # for the branch protection rules. Should be removed at some point.
+    name: build-apps (${{ matrix.version }}, ${{ matrix.platform }})
```

Comment:
Any reason not to remove this now? I can update the rules in order to merge this.

Reply:
Fine by me :) Pushed